src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -2186,42 +2191,32 @@
       text: |
           az group create -g MyResourceGroup --location westeurope
 
-          az network vnet create -g MyResourceGroup -n MyVirtualNetwork \\
-            --address-prefixes 10.0.0.0/16 --subnet-name MyAseSubnet --subnet-prefixes 10.0.0.0/24
+          az network vnet create -g MyResourceGroup -n MyVirtualNetwork --address-prefixes 10.0.0.0/16 --subnet-name MyAseSubnet --subnet-prefixes 10.0.0.0/24
```

Comment:
I'd like to ask why you delete the line break

Reply:
Thought it would make copy/paste experience better, but it did not, so I added them back in.